src/Raven.Studio/typescript/common/notifications/models/operation.ts

```diff
@@ -15,11 +15,19 @@ class operation extends abstractNotification {
     taskType = ko.observable<Raven.Server.Documents.Operations.Operations.OperationType>();
     detailedDescription = ko.observable<Raven.Client.Documents.Operations.IOperationDetailedDescription>();
     
-    startTime = ko.observable<moment.Moment>();
     endTime = ko.observable<moment.Moment>();
-    duration: KnockoutComputed<string>;
-    durationInSeconds: KnockoutComputed<number>;
 
+    startTimeForOperation = ko.observable<moment.Moment>();
```

Comment:
you added start time for every operation type, but here we repeat this for every type, which looks weird. Can we put start time to proper location: single line object? Instead of creating start time/ duration/ etc for documents/revisions/counters/time series/etc?

Reply:
I think it should be: `smugglerListItem`

Reply:
ok - simplified logic